data-serving/scripts/data-pipeline/README.md

```diff
@@ -0,0 +1,18 @@
+# Full data pipeline
+
+Ever wanted to not only convert *or* ingest the line-list data, but do both, together?
+
+Have you ever wanted to import the freshest possible data directly into a
+database?
+
+This script is for you.
+
+## How to run it
+
+```shell
+./run_pipeline.sh [-m <mongodb_connection_string>] [-d <database>] [-c <collection>] [-r <sample_rate>]
```

Comment:
can we name that script with what it does? "running a pipeline" is open to interpretation


![image](https://user-images.githubusercontent.com/1255432/83911541-bc8ec080-a76c-11ea-9fc8-1cee5d398273.png)

I suggest
fetch_latest_data_convert_data_setup_db_import_data_cleanup.sh

Reply:
hah so you DO see why I named it that!

how about... convert_and_import_latest_data.sh?

Reply:
I like that proposal